app/models/user.rb

```diff
@@ -39,6 +40,10 @@ def admin?
     role.eql? 'admin'
   end
 
+  def generate_slug
+    update_columns(slug: generate_token)
```

Comment:
Lets use `update_column` here. Also instead of getting random token, we can use user `first_name` as slug.

Reply:
discussed